editor/plugins/tiles/tile_atlas_view.cpp

```diff
@@ -39,6 +39,8 @@
 #include "scene/gui/label.h"
 #include "scene/gui/panel.h"
 #include "scene/gui/view_panner.h"
+#include <scene/resources/style_box_flat.h>
+#include <scene/theme/default_theme.cpp>
```

Comment:
```suggestion
#include "scene/resources/style_box_flat.h"
```
You don't include the default theme directly, and use `"` for includes